src/components/common/ColonyHome/ColonyUnclaimedTransfers/ColonyUnclaimedTransfers.tsx

```diff
@@ -59,6 +64,11 @@ const ColonyUnclaimedTransfers = () => {
    */
   const token = firstItem?.token;
 
+  const handleClaimSuccess = () => {
+    startPollingColony(1_000);
+    setTimeout(stopPollingColony, 10_000);
```

Comment:
These are currently "magic" values. There's definitely a good first issue in moving all such "magic" values to our constants file.